docs/Makefile

```
@@ -35,11 +35,17 @@ html:
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
 doctest:
```

Dependencies are your friend!
```suggestion
doctest: doctest-rust doctest-python
```